spec/discovery_dispatcher/delete_from_all_indexes_job_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe DiscoveryDispatcher::DeleteFromAllIndexesJob do
+  describe '.perform' do
+    it 'performs the indexing process successfully' do
+      stub_request(:delete, 'http://www.example-indexer.com/items/xz404nk7341').
+        to_return(status: 200)
+      index_job = described_class.new('delete', 'xz404nk7341', 'searchworkspreview')
+      expect(Rails.logger).to receive(:debug).with(/Processing/).and_call_original
```

Comment:
i find it confusing that you're passing in `searchworkspreview` to a delete all case.